Formula/apache-drill.rb

```diff
@@ -1,9 +1,8 @@
 class ApacheDrill < Formula
   desc "Schema-free SQL query engine for Hadoop and NoSQL"
   homepage "https://drill.apache.org/download/"
-  url "https://www.apache.org/dyn/closer.cgi?path=drill/drill-1.7.0/apache-drill-1.7.0.tar.gz"
-  mirror "https://archive.apache.org/dist/drill/drill-1.7.0/apache-drill-1.7.0.tar.gz"
```

Comment:
Hi All,

Previous contributor to the apache drill formula here. Unfortunately I can not see why this failed as the failure report seems to have been removed.

As an FYI I believe HomeBrew supports Apache mirror pages so that closer.cgi url should be fine:

https://github.com/Homebrew/legacy-homebrew/issues/39935

https://github.com/Homebrew/brew/blob/master/Library/Homebrew/download_strategy.rb#L422

I would keep the urls as they were but bumping the version to 1.9.0.

We also appear to have another pull request for this update:

https://github.com/Homebrew/homebrew-core/pull/7594

Would be good to mark that one as closed once we get this merged.

I am not too sure how I can contribute to this pull request without making another one?